multimodal/tests/unittests/predictor/test_predictor.py

```diff
@@ -456,41 +456,29 @@ def test_model_configs():
                 "hf_text",
                 "timm_image",
                 "clip",
-                "categorical_transformer",
-                "numerical_transformer",
+                "ft_transformer",
                 "fusion_transformer",
             ],
-            "categorical_transformer": {
-                "out_features": 192,
+            "ft_transformer": {
+                "data_types": ["numerical", "categorical"],
+                "embedding_arch": ["linear"],
                 "d_token": 192,
-                "num_trans_blocks": 0,
-                "num_attn_heads": 4,
```

Comment:
Looks like `num_attn_heads ` is removed from config?

Reply:
`num_attn_heads` is for categorical_transformer.